opr-core/src/model/resharechains.ts

```diff
@@ -0,0 +1,59 @@
+import {DecodedReshareChain} from 'opr-models';
```

Comment:
🎶 I'm looking for my license header
It's been a few days now
I've looked high and low
But I can't find it anywhere

Oh, license header
Where could you be?
I need you to complete my code
But you're nowhere to be found

I've checked all my usual places
But you're just not there
So if you see my license header
Please let me know

I'm getting kind of worried
I hope you're not lost forever
But if you are, I'll just use someone else's
Until you turn up again 🎶

(Hit song lyrics courtesy of GPT-3)

Reply:
Adorable! Done.